tests/wpt/mozilla/meta/mozilla/preferences.html.ini

```diff
@@ -1,10 +1,3 @@
 [preferences.html]
   type: testharness
-  prefs: [
-    dom.testbinding.preference_value.falsy:false,
-    dom.testbinding.preference_value.truthy:true,
-    dom.testbinding.preference_value.string_empty:,
-    dom.testbinding.preference_value.string_test:test,
-    dom.testbinding.preference_value.space_string_test:test1 test2,
-    dom.testbinding.preference_value.quote_string_test:"test1 test2",
-    ]
+  prefs: [dom.testbinding.preference_value.falsy:false, dom.testbinding.preference_value.truthy:true, dom.testbinding.preference_value.string_empty:, dom.testbinding.preference_value.string_test:test, dom.testbinding.preference_value.space_string_test:test1 test2, dom.testbinding.preference_value.quote_string_test:"test1 test2"]
```

Comment:
Running `./mach update-wpt` has some uncommitted changes that result in various formatting updates. This is one of them, though they should probably happen in a different PR.